[SEA-2023] venue updates and enable sponsorship

harbinger55 opened the pull request.

MikeRosTX commented on static/events/2023-seattle/Prospectus-2023.pdf:
RENAME to all lower case

*FROM*  **_static/events/2023-seattle/Prospectus-2023.pdf_** 

*TO*  **_static/events/2023-seattle/prospectus-2023.pdf_**

Then re-upload the PDF.

mattstratton replied:
actually please remove this file from this PR and instead upload it (and then link to it) in the devopsdays/devopsdays-assets repo (we try to avoid having larger files in this repo)

harbinger55 commented:
Sorry for the delay. I have moved the prospectus to devopsdays-assets in PR https://github.com/devopsdays/devopsdays-assets/pull/153 and updated the pages appropriately.
